bin/event_stream_pipe.c

```diff
@@ -18,6 +18,10 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+#ifdef _MSC_VER
+#pragma warning (disable: 4996 4706)
```

Comment:
4996 is about fopen/sprintf vs fopen_s/sprintf_s
4706 is about assignment expression in condition clause like `while( a = b))`